test/Table_Tests/src/In_Memory/Main.enso

```diff
@@ -22,5 +23,6 @@ spec =
     Aggregate_Column_Spec.spec
     Builders_Spec.spec
     Join_Performance_Spec.spec
+    Split_Tokenize_Spec.spec
```

Comment:
Hmm, if we plan to also have this functionality on DB side, I'm wondering if it won't be better to put it into `Common_Table_Operations` and just disable tests on DB by a flag.

But I guess it's fine to keep it here and move when needed.